engine/CMakeLists.txt

```diff
@@ -861,7 +861,7 @@ SET(LIBCMD_SOURCES
     src/cmd/weapon_info.cpp
     src/cmd/mount_size.cpp
     src/cmd/weapon_type.cpp
-)
+        src/vega_py_run.cpp src/vega_py_run.hpp)
```

Comment:
Headers aren't typically included in these lists. Instead, we have INCLUDE_DIRECTORIES at around line 157. Also, we usually use h files.

Reply:
We're pretty much a C++ project, and others don't include our files so whether we use `.hpp`, `.hxx`, `.h`, etc doesn't really matter; we should, however, remain consistent so for the time being we should be using `.h` files.

As I noted in https://github.com/isocpp/CppCoreGuidelines/issues/1965#issuecomment-1218460543, if we need to block C or C++ we can do so through pragma directives; we shouldn't here since we're entirely C/C++ and don't compile them as pure C or pure C++.

For our purposes here, consistency is more key.